### PR TITLE
added more exercises about probability

### PR DESCRIPTION
**Pull request by 1apple7seeds**
- See file python_for_probability_bonus_level.ipynb

### CODE REVIEW

File: exercises/python/python_for_probability_bonus_level.ipynb [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": []
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Exercise 4: Joint Probability Distributions [0.25 points]\n",
```
Comment (alex-mitrevski) [inline exercises/python/python_for_probability_bonus_level.ipynb]: Exercise 4 is before exercise 3?

Comment (1apple7seeds) [reply exercises/python/python_for_probability_bonus_level.ipynb]: you right! I changed the location of the last 2 problems and forgot to change the numbers.
Thanks

**Comment by Sushant-Chavan** [comment]
@1apple7seeds Will this be repeated in the python for probability session which is scheduled for Iswariya on the 22nd of March? Since the intro to python session will not be done by the time you complete yours, maybe it will be difficult for the students to solve the programming exercises. Probably just sync up your plans with her. :) 

Also Nga assumed that there wont be any programming during the Mathematics lectures and informed the students that laptops are not mandatory for these days. If you need them to get their laptop, just inform them about it on Slack.

**Comment by 1apple7seeds** [comment]
- Removed the problems that required programming from within the file.
- Named the file probability_warmup.
- Relocated it inside contents/math/.

I hope this is ok :)

**Pull request merged**